src/visualizers/VisualizerGrid.ts

```diff
@@ -0,0 +1,1214 @@
+                dependentValue !== Property.None,
+        },
+        property10Visualization: {
+            value: this.property10Visualization,
+            from: PropertyVisualization,
+            displayName: 'Property 10 Visualization',
+            required: false,
+            visibleDependency: 'property10',
+            visiblePredicate: (dependentValue: Property) =>
+                dependentValue !== Property.None,
+        },
+        property10MainColor: {
+            value: this.property10MainColor,
+            forceType: 'color',
+            displayName: 'Property 10 color:',
+            required: false,
+            visibleDependency: 'property10Visualization',
+            visibleValue: PropertyVisualization.Color,
+        },
+        property11: {
+            value: this.property11,
+            from: Property,
+            displayName: 'Property 11',
+            required: false,
+            visibleDependency: 'property10',
+            visiblePredicate: (dependentValue: Property) =>
+                dependentValue !== Property.None,
+        },
+        property11Visualization: {
+            value: this.property11Visualization,
+            from: PropertyVisualization,
+            displayName: 'Property 11 Visualization',
+            required: false,
+            visibleDependency: 'property11',
+            visiblePredicate: (dependentValue: Property) =>
+                dependentValue !== Property.None,
+        },
+        property11MainColor: {
+            value: this.property11MainColor,
+            forceType: 'color',
+            displayName: 'Property 11 color:',
+            required: false,
+            visibleDependency: 'property11Visualization',
+            visibleValue: PropertyVisualization.Color,
+        },
+        property12: {
+            value: this.property12,
+            from: Property,
+            displayName: 'Property 12',
+            required: false,
+            visibleDependency: 'property11',
+            visiblePredicate: (dependentValue: Property) =>
+                dependentValue !== Property.None,
+        },
+        property12Visualization: {
+            value: this.property12Visualization,
+            from: PropertyVisualization,
+            displayName: 'Property 12 Visualization',
+            required: false,
+            visibleDependency: 'property12',
+            visiblePredicate: (dependentValue: Property) =>
+                dependentValue !== Property.None,
+        },
+        property12MainColor: {
+            value: this.property12MainColor,
+            forceType: 'color',
+            displayName: 'Property 12 color:',
+            required: false,
+            visibleDependency: 'property12Visualization',
+            visibleValue: PropertyVisualization.Color,
+        },
+    }
+
+    checkParameters() {
+        const status = super.checkParameters()
+        if (typeof this.message !== 'string') {
```

Comment:
This isn't doing anything since `this.message` is always a string.

Reply:
I think I need to remove this.message since it is not in the UI.

Reply:
Sounds good. I will resolve this once you remove it.